src/index.js

```diff
@@ -1,17 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Calculator from './components/Calculator';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
   </React.StrictMode>,
   document.getElementById('root'),
 );
```

Comment:
- [ ] <b>REQUIRED:</b> As mentioned in the previous review, `Calculator` should not be imported and used in here, it should be imported and used in `App` component. Kindly create `App` component in `/src` directory and then import and use `Calculator` component in there. Furthermore, please make sure that the `App` component is also class-based component.